util/topgen/templates/chiplevel.sv.tpl

```diff
@@ -816,25 +800,25 @@ module chip_${top["name"]}_${target["name"]} (
     .Ast2PadOutWidth(ast_pkg::Ast2PadOutWidth),
     .Pad2AstInWidth(ast_pkg::Pad2AstInWidth)
   ) u_ast (
+    // clocks and resets supplied for detection
+    .gd_clks_i      ( clkmgr_aon_clocks    ),
+    .gd_rsts_i      ( rstmgr_aon_resets    ),
+    .gd_ext_clk_i   ( sck_monitor          ),
     // tlul
     .tl_i                  ( base_ast_bus ),
     .tl_o                  ( ast_base_bus ),
     // init done indication
     .ast_init_done_o       ( ast_init_done ),
     // buffered clocks & resets
-    // Reset domain connection is manual at the moment
-    .clk_ast_adc_i         ( clks_ast.clk_ast_adc_ctrl_aon_io_div4_peri ),
-    .rst_ast_adc_ni        ( rsts_ast.rst_ast_adc_ctrl_aon_sys_io_div4_n[DomainAonSel] ),
-    .clk_ast_alert_i       ( clks_ast.clk_ast_sensor_ctrl_aon_io_div4_secure ),
-    .rst_ast_alert_ni      ( rsts_ast.rst_ast_sensor_ctrl_aon_lc_io_div4_n[DomainAonSel] ),
-    .clk_ast_es_i          ( clks_ast.clk_ast_edn0_main_secure ),
-    .rst_ast_es_ni         ( rsts_ast.rst_ast_edn0_sys_n[Domain0Sel] ),
-    .clk_ast_rng_i         ( clks_ast.clk_ast_entropy_src_main_secure ),
-    .rst_ast_rng_ni        ( rsts_ast.rst_ast_entropy_src_sys_n[Domain0Sel] ),
-    .clk_ast_tlul_i        ( clks_ast.clk_ast_ast_io_div4_secure ),
-    .rst_ast_tlul_ni       ( rsts_ast.rst_ast_ast_lc_io_div4_n[DomainAonSel] ),
-    .clk_ast_usb_i         ( clks_ast.clk_ast_usbdev_usb_peri ),
-    .rst_ast_usb_ni        ( rsts_ast.rst_ast_usbdev_usbif_n[Domain0Sel] ),
+    // Due to the fact that comportable ips do not support split domains,
+    // the reset power domain connection below is incorrect
+    // This is TODO
```

Comment:
nit: can we make an issue for this? does this relate to #7978?

Reply:
No it's related to #7980 which has already been merged, which means this
comment is out of date.
I'll fix that.

Reply:
Thanks!